collect_app/src/main/java/org/odk/collect/android/utilities/MediaUtils.java

```diff
@@ -619,6 +619,7 @@ public static boolean isGooglePhotosUri(Uri uri) {
      */
     public static boolean isGoogleDriveDocument(Uri uri) {
         return "com.google.android.apps.docs.storage".equals(uri.getAuthority())
+                || "com.google.android.apps.docs.storage.legacy".equals(uri.getAuthority())
                 || uri.getAuthority().startsWith("com.google.android.apps.photos.content");
```

Comment:
I found that there are two "google drive" tab in my file manager , and their returned  Uri's authorities are different ,so i add
 "com.google.android.apps.docs.storage.legacy"  for supporting legacy Google Drive File.